_posts/2023-07-11-nvidia-is-a-new-sponsor.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "NVIDIA Is A New Sponsor Of The Scikit-Learn consortium at the Inria Foundation"
+date: July 11, 2023
+
+categories:
+  - Funding
+tags:
+  - Sponsor
+
+featured-image: NVIDIAxsklearn.jpg
+
+postauthors:
+  - name: NVIDIA
+    website: https://nvidia.com/
+    image: "nvidia-logo.png"
+  - name: François Goupil
+    email: francois.goupil@inria.fr
+    website: https://github.com/francoisgoupil
+    image: "francois_goupil.jpeg"
+---
+<div>
+  <img src="/assets/images/posts_images/{{ page.featured-image }}" alt="">
+  {% include postauthor.html %}
+</div>
+
+<span style="color:red">*Sponsored blog post* </span>
+
+We are thrilled to announce that [NVIDIA](https://www.nvidia.com) has joined the [scikit-learn consortium](https://scikit-learn.fondation-inria.fr/) as a corporate partner. As a leading provider of GPU-accelerated computing solutions, we at NVIDIA recognize the importance of machine learning and the role it plays in the growth of many industries and areas of science. Our partnership with the scikit-learn consortium demonstrates our commitment to supporting the development and advancement of open-source software in the machine learning community.
```

Comment:
Please you add this (https://developer.nvidia.com/open-source) hyper link to "open-source software" statement.